src/elements/polymer-helmet.html

```diff
@@ -0,0 +1,95 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<dom-module id="polymer-helmet">
+  <template></template>
+
+  <script>
+    /* eslint-disable max-len */
+    class PolymerHelmet extends Polymer.Element {
+      static get is() {
+        return 'polymer-helmet';
+      }
+
+      static get properties() {
+        return {
+          active: Boolean,
+          title: String,
+          description: String,
+          image: String,
+          label1: String,
+          data1: String,
+          label2: String,
+          data2: String,
+          _defaultTitle: {
+            type: String,
+            value: '{$ title $}',
+          },
+          _defaultDescription: {
+            type: String,
+            value: '{$ description $}',
+          },
+          _defaultImage: {
+            type: String,
+            value: '{$ image if image.startsWith("http") else (url + image) $}',
+          },
+          _defaultLabel1: {
+            type: String,
+            value: '{$ location.label $}',
+          },
+          _defaultData1: {
+            type: String,
+            value: '{$ location.name $}',
+          },
+          _defaultLabel2: {
+            type: String,
+            value: '',
+          },
+          _defaultData2: {
+            type: String,
+            value: '',
+          },
+        };
+      }
+
+      static get observers() {
+        return [
+          '_setMetaInfo(active, title, description, image, label1, data1, label2, data2)',
+        ];
+      }
+
+      _setMetaInfo(active, title, description, image, label1, data1, label2, data2) {
+        if (active) {
```

Comment:
you can add at the beginning

title = title || this._defaultTitle;
description = description || this._defaultDescription;

....

So you wouldn't repeat this checks